package-lock.json

```diff
@@ -67,6 +67,12 @@
         }
       }
     },
+    "@babel/parser": {
```

Comment:
Commiting the package-lock is a different task, isn't it ?
Is this really a requirement for this PR ?